### PR TITLE
updated login flow

### PR DESCRIPTION
**Pull request by sohilpandya**
- check if user is primary and has no mission_statement
- then redirect user to the edit page otherwise redirect to walkthrough page

related to #787

### CODE REVIEW

File: server/handlers/auth/login.js [hunk 1]
```diff
@@ -46,7 +47,21 @@ module.exports = function (request, reply) {
...
+
+        if (person.user_type === 'admin') {
+          return reply.redirect('/').state('token', token);
+        }
+        request.server.methods.pg.organisations.getDetails(orgId, function (pgError, orgData) {
+          Hoek.assert(!pgError, 'db error');
+          var missionStatement = orgData.org.mission_statement;
+
+          if(person.user_type === "primary" && !missionStatement) {
+            return reply.redirect('/orgs/' + orgId + '/edit').state('token', token);
```
Comment (Jbarget) [inline server/handlers/auth/login.js]: would we need to give the user an idea of why they're being redirected to the organisation edit page?

File: server/handlers/auth/login.js [hunk 2]
```diff
@@ -46,7 +47,21 @@ module.exports = function (request, reply) {
           return reply.view('login', {error: { message: 'Sorry, something went wrong. Please try again.', values: request.payload } }).code(500);
         }
         var token = jwt.sign(session, process.env.JWT_SECRET);
-        return reply.redirect('/').state('token', token);
+        var orgId = person.org_id
```
Comment (Jbarget) [inline server/handlers/auth/login.js]: semi-colon here

Comment (sohilpandya) [inline server/handlers/auth/login.js]: pushed this change up. Thanks!

File: server/handlers/auth/login.js (hunk 1 above)
Comment (SimonLab) [inline server/handlers/auth/login.js]: good question, maybe we can display a message at the top of the page.

Comment (sohilpandya) [inline server/handlers/auth/login.js]: @Jbarget can we merge this in? We've started off the discussion related to what should happen when the user comes onto this page and we can take action based on the outcome of that discussion.

Comment (Jbarget) [inline server/handlers/auth/login.js]: sure thing

**Pull request merged**